src/lib/snack-bar/snack-bar.ts

```diff
@@ -64,6 +62,10 @@ export class MdSnackBar {
     } else {
       snackBarRef.containerInstance.enter();
     }
+    if (config.autoHide > 0) {
+      setTimeout(() => snackBarRef.dismiss(), config.autoHide + 225);
```

Comment:
Forgot I had this in there.

Yes, the 225 is for the animation.  Created a todo to handle the case I was looking at.